src/fflib/AFunction.cpp

```diff
@@ -1241,7 +1241,10 @@ void Init_map_type()
        );
 
      TheOperators->Add("%",
-       new OneBinaryOperator<Op2_mod<long,long,long> >
+       new OneBinaryOperator<Op2_mod<long,long,long> >,
+       new OneOperator2<double,double,double>(fmod),
+       new OneOperator2<double,long,double>(fmod),
+       new OneOperator2<double,double,long>(fmod)
```

Comment:
```suggestion
       new OneOperator2<double,double,double>(fmod)
```
I don't think it's necessary to add these casts.